lib/ui/app_bar.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+
+// We define here generic margins for the app
+
+abstract class CustomAppBar {
+  static AppBar build(int currentHomeTab) {
+    switch (currentHomeTab) {
+      case 0: // MyPlot
+        return buildForHome(Strings.myPlotTab, profileAction(), homeIcon() );
+        break;
+      case 1: // Profit / Loss
+        return buildWithTitle(Strings.profitLossTab);
+        break;
+      case 2: // Articles
+        return buildWithTitle(Strings.discoverTab);
+        break;
+      case 3: // Community
```

Comment:
Here is another place where we would be able to use the const values for the tabs